components/tools/OmeroWeb/omeroweb/webgateway/marshal.py

```diff
@@ -134,9 +134,18 @@ def imageMarshal(image, key=None):
 
     # big images
     tiles = image._re.requiresPixelsPyramid()
-    width, height = image._re.getTileSize()
-    levels = image._re.getResolutionLevels()
-    zoomLevelScaling = image.getZoomLevelScaling()
+    rv['tiles'] = tiles
+    if (tiles):
```

Comment:
I am asking about all of the attributes below, they were in try catch block. Are you sure it is not needed anymore?